pkg/northbound/e2/e2.go

```diff
@@ -15,8 +15,10 @@
 package e2
 
 import (
-	sb "github.com/onosproject/ran-simulator/api/e2"
+	"github.com/onosproject/ran-simulator/api/e2"
+	pb "github.com/onosproject/ran-simulator/api/e2"
```

Comment:
the `pb` import is just a repeat of the one above it - I thin `e2` is fine

Reply:
fixed